src/styles/button.css

```diff
@@ -0,0 +1,23 @@
+.card__button {
+  margin-top: 16px;
+}
+
+.card__button_text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--color-text-button);
+  font-weight: 700;
+  font-size: var(-size-text-big);
+  text-decoration: none;
+  width: 166px;
+  height: 40px;
+  background-color: var(--color-button);
+  border-radius: 5px;
+  border: 1px solid var(--color-button);
+}
+
+.card__button:hover .card__button_text:hover {
+  background-color: var(--color-text-button);
+  color: var(--color-button);
+}
```

Comment:
it's not BEM block, it's related to card

Reply:
done